apps/music_proxy.cpp

```diff
@@ -39,13 +39,17 @@
 
 namespace po = boost::program_options;
 
+namespace
+{
+const double defaultMusicTimestep = 0.0001;
+const brion::URI pluginURI( MONSTEER_BRION_SPIKES_PLUGIN_SCHEME + "://" );
```

Comment:
I have no idea if they should(?), but I can for sure do it. Can anyone else confirm if it's part of our coding standards?